eva-accession-deprecate/src/test/java/uk/ac/ebi/eva/accession/deprecate/test/configuration/MongoTestConfiguration.java

```diff
@@ -0,0 +1,35 @@
+/*
+ *
+ * Copyright 2019 EMBL - European Bioinformatics Institute
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ *
+ */
+package uk.ac.ebi.eva.accession.deprecate.test.configuration;
+
+import org.springframework.boot.autoconfigure.domain.EntityScan;
+import org.springframework.boot.test.autoconfigure.data.mongo.AutoConfigureDataMongo;
+import org.springframework.context.annotation.Configuration;
+import org.springframework.data.mongodb.config.EnableMongoAuditing;
+import org.springframework.data.mongodb.repository.config.EnableMongoRepositories;
+
+@Configuration
+@EntityScan(basePackages = {"uk.ac.ebi.eva.accession.core.persistence"})
+@EnableMongoRepositories(basePackages = {
+        "uk.ac.ebi.eva.accession.core.persistence",
+        "uk.ac.ebi.ampt2d.commons.accession.persistence.mongodb.repository"})
+@EnableMongoAuditing
```

Comment:
I am curious as to why we need auditing for test data for which we are using fake data anyway.

Reply:
I think this was for filling in the date timestamps